_posts/2021/2021-12-06-tidy-up-your-routes-with-only.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Tidy Up Your Routes with Only"
+description: "Tidy routes are happy routes"
+layout: article
+category: ruby
+image:
+  base: "2021/tidy-up-your-routes-with-only"
+  alt: "Road Closed. Diversion. Road signs."
+  credit: "Ben Wicks"
+  source: "https://unsplash.com/photos/zwN1MwCtR5Y"
+
+---
+
+It is easy with [Rails’ syntax for defining routes](https://guides.rubyonrails.org/routing.html) to open up more URLs in your application than you might intend.
```

Comment:
*make available more URLs in your application than you might intend to (?)

Reply:
```suggestion
It is easy with [Rails’ syntax for defining routes](https://guides.rubyonrails.org/routing.html) to make more URLs in your application available than you might intend to.
```